File: CleanDB.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.Statement;

public class CleanDB {
    public static void main(String[] args) throws Exception {
        String url = "jdbc:postgresql://ep-calm-sky-az2uy53g-pooler.c-3.ap-southeast-1.aws.neon.tech/neondb?sslmode=require";
        String user = "neondb_owner";
        String password = "npg_kt3yHwMW7RQi";
        
        System.out.println("Connecting to database...");
        try (Connection conn = DriverManager.getConnection(url, user, password);
             Statement stmt = conn.createStatement()) {
            System.out.println("Dropping schema public...");
            stmt.execute("DROP SCHEMA public CASCADE;");
            System.out.println("Creating schema public...");
            stmt.execute("CREATE SCHEMA public;");
            System.out.println("Done! Database is clean.");
        }
    }
}

File: admin-app/clean_db.js
const { Client } = require('pg');
const client = new Client({
  connectionString: 'postgresql://neondb_owner:npg_kt3yHwMW7RQi@ep-calm-sky-az2uy53g-pooler.c-3.ap-southeast-1.aws.neon.tech/neondb?sslmode=require'
});
client.connect().then(() => {
  return client.query('DROP SCHEMA public CASCADE; CREATE SCHEMA public;');
}).then(() => {
  console.log('Database cleaned successfully.');
  return client.end();
}).catch(console.error);
